ffs/integrationtest/integration_test.go

```diff
@@ -488,6 +488,38 @@ func TestFilecoinCountryFilter(t *testing.T) {
 	require.Equal(t, p.Miner, "t01001")
 }
 
+func TestFilecoinMaxPriceFilter(t *testing.T) {
```

Comment:
This just verifies that we fail to find miners if our max price is too low, then verifies we find the miner if our max price is high enough.